cmd/opampsupervisor/supervisor/config/config.go

```diff
@@ -266,6 +271,17 @@ type Telemetry struct {
 	Resource map[string]*string `mapstructure:"resource"`
 }
 
+type HealthCheck struct {
+	Port int `mapstructure:"port"`
```

Comment:
Sorry, probably should have taken a closer look and called this out earlier: it may be overkill, but I think we should use [confighttp.ServerConfig](https://pkg.go.dev/go.opentelemetry.io/collector/config/confighttp#ServerConfig) here. It keeps us consistent with Collector receivers, the healthcheck(v2) extension, and offers all the options we could need out of the box.

Reply:
Working on it.